MIDAS/.gitignore

```diff
@@ -1,2 +1,6 @@
+# User configs
 .pio
 .vscode
+
+# Mac garbage
+.DS_Store
```

Comment:
would it make sense to make this `**/.DS_Store`? for pysim repo they keep popping up in every subdirectory